x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -7,95 +7,96 @@
 
 /* eslint-disable react-hooks/exhaustive-deps */
 
-import { i18n } from '@kbn/i18n';
```

Comment:
Are these changes until line ~100 some kind of sorting, a linter rule or something on your end? I guess we should ignore it, right? something important we should know?

Reply:
yeah my habit of running "Organize import" every time I save... I should have not done that on this file, apologies for this

Reply:
np